package.json

```diff
@@ -25,7 +25,6 @@
   "license": "MIT",
   "devDependencies": {
     "bpmn-moddle": "^7.1.3",
-    "bpmnlint": "^7.8.0",
```

Comment:
Is it intentional to move this to a production dependency?

Reply:
Ooops!

Reply:
Fixed!